_search-plugins/star-tree-index.md

```diff
@@ -39,26 +39,26 @@ The following image illustrates a standard star-tree index structure.
 
 <img src="{{site.url}}{{site.baseurl}}/images/star-tree-index.png" alt="A star-tree index containing two dimensions and two metrics" width="700">
 
-Sorted and aggregated star-tree documents are backed by `doc_values` in an index. `doc_values` use the following pattern:
+Sorted and aggregated STIX documents are backed by `doc_values` in an index. The columnar data found in `doc_values` are stored using the following properties:
 
-- The values are sorted based on the order of their `ordered_dimension`. In the preceding image, the dimensions are determined by the `status` setting and then by the `port` for each status.
+- The values are sorted based on the fields set in the `ordered_dimension` setting. The first field configured in `ordered_dimensions` are considered primary fields, and any corresponding fields of the primary are considered secondary.  In the preceding image, the dimensions are determined by the `status` setting and then by the `port` for each status.
 - For each unique dimension/value combination, the aggregated values for all the metrics, such as `avg(size)` and `count(requests)`, are precomputed during ingestion.
 
 ### Leaf nodes
 
-Each node in a star-tree index points to a range of star-tree documents. Nodes can be further split into child nodes based on the [max_leaf_docs configuration]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/star-tree/#star-tree-configuration-parameters). The number of documents that a leaf node points to is less than or equal to the number set by `max_leaf_docs`. This ensures that the maximum number of documents that need to traverse nodes to derive an aggregated value is at most the number of `max_leaf_docs`, which provides predictable latency.
+Each node in a star-tree index points to a range of star-tree documents. Nodes can be further split into child nodes based on the [max_leaf_docs configuration]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/star-tree/#stix-configuration-options). The number of documents that a leaf node points to is less than or equal to the number set by `max_leaf_docs`. This ensures that the maximum number of documents that need to traverse nodes to derive an aggregated value is at most the number of `max_leaf_docs`, which provides predictable latency.
```

Comment:
The number of documents that a leaf node points to is less than or equal to value set by `max_leaf_docs`. This ensures that the maximum number of documents that gets traversed to get to the aggregated value is at most the number of `max_leaf_docs`, thus providing predictable latency.

Reply:
You want to say "value" instead of "number"?

Reply:
yes value to me sounds clearer